packages/realm/src/app-services/User.ts

```diff
@@ -152,7 +152,7 @@ export class User<
    * @returns The user ID as a string.
    */
   get id(): string {
-    return this.internal.identity;
+    return this.internal.userId;
```

Comment:
I've added https://github.com/realm/realm-core/pull/7300/files#r1579400825

Reply:
Good question but will not have any impact for users of the SDK if we decide to change `user_id` to `id`.